src/Sample.Worker/Program.cs

```diff
@@ -29,6 +31,10 @@ public static IHostBuilder CreateHostBuilder(string[] args)
 
                         x.AddConsumer<SubmitOrderConsumer>();
                         x.AddConsumer<OrderSubmittedConsumer>();
+                        x.AddConsumer<OrderShippedConsumer>(e =>
+                        {
+                            e.UseTimeout(c => c.Timeout = TimeSpan.FromSeconds(10));
```

Comment:
This isn't really used and probably just confuses people.

Reply:
Won't it fail the SB and not ack of the message handling takes to long?
Ie it will ensure failed process are put to failed?

But you are right it can be found easy enough, not needed here